client/dashboard/domains/name-servers/use-upsell-nudge.ts

```diff
@@ -0,0 +1,21 @@
+import { Domain } from '../../data/domain';
+
+/**
+ * Hook to determine if the upsell nudge should be shown for a domain
+ * @param domain The domain to check
+ * @returns boolean indicating if the upsell nudge should be shown
+ */
+export const useUpsellNudge = ( domain: Domain ): boolean => {
```

Comment:
I wonder why this is a hook? There is no hook-related logic inside.

Reply:
Firstly, I query the domain within the hook, and then I realise that I probably don't need to do it since the consumer/parent component already has the whole domain object to pass.

I can move the method to the `utils.ts` file, but I think it can't hurt to leave it as it is. Maybe we will add some more logic here and become a real hook.

I don't know, what do you think?

Reply:
Done 👍